ui/src/app/views/workspace-about.tsx

```diff
@@ -2,151 +2,276 @@ import {Component} from '@angular/core';
 import * as fp from 'lodash/fp';
 import * as React from 'react';
 
-import {Button, Clickable} from 'app/components/buttons';
-import {FadeBox} from 'app/components/containers';
-import {TwoColPaddedTable} from 'app/components/tables';
-import {EditComponentReact} from 'app/icons/edit';
-import {workspacesApi} from 'app/services/swagger-fetch-clients';
-import colors from 'app/styles/colors';
-import {reactStyles, ReactWrapperBase, withCurrentWorkspace, withUserProfile} from 'app/utils';
-import {sliceByHalfLength} from 'app/utils/index';
-import {navigate} from 'app/utils/navigation';
+import {cdrVersionStore, currentWorkspaceStore, serverConfigStore} from 'app/utils/navigation';
 import {WorkspaceData} from 'app/utils/workspace-data';
-import {WorkspacePermissions} from 'app/utils/workspace-permissions';
-import {Authority, Profile, SpecificPopulationEnum} from 'generated/fetch';
-import {ResearchPurposeDescription, ResearchPurposeItems, specificPopulations} from './workspace-edit';
+
+import {profileApi, workspacesApi} from 'app/services/swagger-fetch-clients';
+
+import {Button, Link} from 'app/components/buttons';
+import {InfoIcon} from 'app/components/icons';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import {TooltipTrigger} from 'app/components/popups';
+import {Spinner} from 'app/components/spinners';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {reactStyles, ReactWrapperBase, withUrlParams, withUserProfile} from 'app/utils';
+import {ResearchPurpose} from 'app/views/research-purpose';
+import {ResetClusterButton} from 'app/views/reset-cluster-button';
+import {CdrVersion, Cohort, Profile, UserRole, WorkspaceAccessLevel} from 'generated/fetch';
+import {WorkspaceShare} from './workspace-share';
```

Comment:
Nit: We should be consistent with view imports. I'd either make this `'app/views/workspace-share'` or change the above imports to be relative. (Preference for the absolute path, over the relative)